luxonis_ml/data/datasets/annotation.py

```diff
@@ -468,7 +461,7 @@ def serialize_path(self, value: FilePath) -> str:
 class DatasetRecord(BaseModelExtraForbid):
     files: Dict[str, FilePath]
     annotation: Optional[Detection] = None
-    task_name: str = Field(validation_alias=AliasChoices("task", "task_name"))
+    task: str
```

Comment:
If I understand correctly now the key is "task" right? If possible I would then internally also stick to this name for the variable names. It gets a bit confusing when sometimes we call it "task" and sometimes "task_name"  (e.g. in `def to parquet_rows()` and also other places)

Reply:
I'm trying to keep this convention:
- `task` (with the single exception in the record for better backwards compatibility) means the full task string: `"detection/segmentation"`
- `task_name` refers only to the first part: `"detection"`